src/tests/ftest/container/snapshot.py

```diff
@@ -389,29 +400,35 @@ def test_snapshots(self):
                 "<<<---" + rand_str(random.randint(1, data_size))
             datasize = len(thedata) + 1
             try:
-                obj, epoch = self.container.write_an_obj(
-                    thedata, datasize, dkey, akey, obj_cls=obj_cls)
+                tx_handle = self.container.get_new_tx()
+                obj = self.container.write_an_obj(thedata,
+                                                  datasize,
+                                                  dkey,
+                                                  akey,
+                                                  obj_cls=obj_cls,
+                                                  txn=tx_handle)
+                self.container.commit_tx(tx_handle)
```

Comment:
same comment here

Reply:
Currently Snapshot.create() Python function is not returning anything and further test is trying to display it for information purpose. So I use the tx_handle to avoid any further changes in the test.
            self.log.info("  ==self.container epoch=     %s", tx_handle)
            self.log.info("  ==snapshot.epoch= %s"
                          , snapshot.epoch)